db/db_wal_test.cc

```diff
@@ -37,7 +37,9 @@ TEST_F(DBWALTest, WAL) {
     writeOpt.disableWAL = true;
     ASSERT_OK(dbfull()->Put(writeOpt, handles_[1], "foo", "v2"));
 
-    ReopenWithColumnFamilies({"default", "pikachu"}, CurrentOptions());
+    Options options = CurrentOptions();
```

Comment:
did adding the check for `kTolerateCorruptedTailRecords` break this test? was it because an SST file had newer data than WAL?